README.md

```diff
@@ -159,6 +159,46 @@ func TestFoo(t *testing.T) {
 }
 ```
 
+Table Driven Tests
+------------------
+Ref: [Issue 223](https://github.com/golang/mock/issues/223)  
+It is important to note, especially when testing multiple cases in a loop, to setup your mock controller within your sub-tests.  
+Otherwise, test may yeild unexpected behavior.
+
+```go
+func TestFoo(t *testing.T) {
+  tests := []struct{
+    name string
+    code int
+    result []byte
+  }{
+    /* test 1 */
+    {"success", 200, []byte(`{"foo":"bar"}`)},
```

Comment:
nit: I think it would help readability to have the labels (e.g., `name`, `code`, `result`)